core/trino-parser/src/main/java/io/trino/sql/tree/ColumnDefinition.java

```diff
@@ -27,25 +27,44 @@ public final class ColumnDefinition
 {
     private final QualifiedName name;
     private final DataType type;
+    private final Optional<Literal> defaultValue;
```

Comment:
This should be an `Optional<Expression>`. Even though we won't initially support it, it can include non-literal expressions such as CURRENT_USER, CURRENT_DATE, etc.